### DISCUSSION TITLE
自己打包了多娜多娜，可惜文件太大了

### DISCUSSION THREAD

**Discussion by zerobikappa** [Gamuxorg/bbs#290] [投递和请求游戏]
游戏名称：多娜多娜 一起来干坏事吧
游戏名称(英文)：
下载地址 :（如果有的话）：

文件大小超过3G了，基本没有网盘可以上传这么大的文件，可以怎么给你们呢？

**Comment by hyh19962008** [comment]
百度云支持单个4G的文件吧，不行用分卷压缩吧

**Comment by zerobikappa** [comment]
好吧，分卷上传了。用的是我自己写的启动脚本，我自己添加了一些可选的额外选项。有需要的话请在后面加 `-h` 或 `--help` 自行查看。暂时只支持wine，我自己运行的话字体有些问题，可能之后我会再试一下添加用proton运行的选项。
https://cloud.189.cn/web/share?code=BJRJbmreqqYv（访问码：e2nj）

**Comment by hyh19962008** [comment]
我这边试了一下（Ubuntu  18.04-64bit），需要先用 -p 参数启动创建cache文件夹才可以成功运行，否则会报kernel32.dll找不到，应该是打包的wine有点问题，大兄弟你再重新看一下？

**Comment by zerobikappa** [comment]
> 我这边试了一下（Ubuntu 18.04-64bit），需要先用 -p 参数启动创建cache文件夹才可以成功运行，否则会报kernel32.dll找不到，应该是打包的wine有点问题，大兄弟你再重新看一下？

有点奇怪……这个选项应该是不相关的，加选项 -p 只是手动在appimage包的同级目录创建一个 .cache 文件夹(重启后不消失)，如果你不创建的话那个文件夹会在 /tmp/ 目录里创建的(重启后会消失)……你试一下把你手动创建的那个和 /tmp/ 里的那个都删掉，不带任何参数再开一次，看一下是不是还是同样的情况？
我没有打包wine进去，用的应该是你自己系统的wine……

**Comment by hyh19962008** [comment]
应该是我这边问题，上传站里了，感谢投稿。
有兴趣的话可以加入我们站开发者群300610898^_-

**Comment by lnxyz** [comment]
你们打包win 游戏 wine完全 不用打包进去，wine 可以直接外挂 appimage 版的wine 如：https://github.com/mmtrt/WINE_AppImage; 这样可以节省不少空间
